src/cli/utils.ts

```diff
@@ -24,7 +24,7 @@ export const processResult = (result: TaskGroupResult) => {
     const { taskName, averageMs, value } = result.map[key];
     return {
       taskName,
-      value: averageMs || Number(value),
+      value: averageMs !== undefined ? averageMs : Number(value),
```

Comment:
what was this one?

Reply:
before I changed it to this, it didn't take interaction test results with value of `0` into account